### PR TITLE
GH-1312 - Update title when running a plugin

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Currently, the page title is "Focalboard". When running as a plugin, the page title should be "Boards - Mattermost" to align with the new branding. Personal Editions should continue to be titled "Focalboard"

This is only displayed when No Board is selected.

#### Ticket Link

  Fixes https://github.com/mattermost/focalboard/issues/1312

Plugin (notice url)
  
Screen Shot 2021-09-22 at 12 30 48 PM

Personal Server (notice url)

Screen Shot 2021-09-22 at 12 30 05 PM

### CODE REVIEW

**Review by hahmadia — APPROVED** [review]

**Comment by ogi-m** [comment]
@sbishel I see the title is "Mattermost - Boards"
title

Shouldn't it be "Boards - Mattermost" ?
I checked Playbooks and that reads "Playbooks - Mattermost"

File: webapp/src/pages/boardPage.tsx [hunk 1]
```diff
@@ -131,6 +131,8 @@ const BoardPage = (props: Props) => {
                 title += ` | ${activeView.title}`
             }
             document.title = title
+        } else if (Utils.isFocalboardPlugin()) {
+            document.title = 'Mattermost - Boards'
```
Comment (hahmadia) [inline webapp/src/pages/boardPage.tsx:135]:
```suggestion
            document.title = 'Boards - Mattermost'
```

**Review by hahmadia — APPROVED** [review]
Address Ogi Change

**Comment by hahmadia** [comment]
@ogi-m 

I helped resolve the comment you pointed out. Feel free to try again.

**Review by ogi-m — APPROVED** [review]
Thanks @hahmadia! LGTM 👍🏻

**Comment by sbishel** [comment]
Thanks @hahmadia for taking care of this.

**Pull request merged**